### PR TITLE
New cleanup feature to support CEPH storage backend

### PR DESCRIPTION
**Pull request by Pansanel**

### CODE REVIEW

**Comment by Mamut3D** [comment]
Thanks for the PR, I will test it

File: cloudkeeper_os/imagemanager.py [hunk 1]
```diff
@@ -88,6 +89,8 @@ def add_appliance(self, appliance):
         properties = utils.extract_appliance_properties(appliance)
         min_ram = int(properties.get("APPLIANCE_RAM", 0))
 
+        properties[IMAGE_STATUS_TAG] = 'ACTIVE'
```
Comment (Mamut3D) [inline cloudkeeper_os/imagemanager.py:92]: I have tried it, however for some reason `APPLIANCE_STATUS` does not seem to be propagating to openstack as an attribute. 
Can you verify that it correctly propagates to openstack?

File: cloudkeeper_os/imagemanager.py [hunk 2]
```diff
@@ -142,13 +145,44 @@ def remove_appliance(self, appliance):
...
         if not glance_image:
-            LOG.info("Cannot delete image: image not found")
+            LOG.info("Cannot mark image for removal: image not found")
             return None
 
-        LOG.info("Deleting image: '%s'" % glance_image.id)
-        glance.images.delete(glance_image.id)
+        LOG.info("Marking image for removal: '%s'" % glance_image.id)
+        properties = {}
+        properties[IMAGE_STATUS_TAG] = 'EOL'
```
Comment (Mamut3D) [inline cloudkeeper_os/imagemanager.py:153]: Same as the above,  can you verify propagation?

**Comment by Pansanel** [comment]
Could you test it?

**Review by Mamut3D — CHANGES_REQUESTED** [review]

File: cloudkeeper_os/imagemanager.py (hunk 2 above)
Comment (Mamut3D) [reply cloudkeeper_os/imagemanager.py:153]: Sorry for the late response, I have tested the code during December, however I have left the review in unsubmitted state by mistake.

Comment (Pansanel) [reply cloudkeeper_os/imagemanager.py]: I have test on my infra, I have the 'APPLIANCE_STATUS' tag set to 'ACTIVE, as requested. Can you run cloudkeeper-os in verbose mode on your side to see what happens?
You can also perform a quick test with this SQL query:
`MariaDB [glance]> select distinct(name) from image_properties;`

**Comment by Pansanel** [comment]
I have test on my infra, I have the 'APPLIANCE_STATUS' tag set to 'ACTIVE, as requested. Can you run cloudkeeper-os in verbose mode on your side to see what happens?
You can also perform a quick test with this SQL query to see if the tag exists:
```MariaDB [glance]> select distinct(name) from image_properties;```

**Comment by Mamut3D** [comment]
I am sorry, I am currently unable to test it further. If it works for you, just merge the request.

**Pull request merged**